app/uk/gov/hmrc/plasticpackagingtax/registration/views/phone_number_page.scala.html

```diff
@@ -15,35 +15,35 @@
  *@
 
 @import uk.gov.hmrc.govukfrontend.views.html.components.GovukInput
-@import uk.gov.hmrc.govukfrontend.views.html.components.GovukButton
 @import uk.gov.hmrc.plasticpackagingtax.registration.views.model.Title
 @import uk.gov.hmrc.plasticpackagingtax.registration.views.model.BackButton
 @import uk.gov.hmrc.plasticpackagingtax.registration.controllers.{routes => pptRoutes}
 @import uk.gov.hmrc.plasticpackagingtax.registration.forms.PhoneNumber
 @import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.sectionHeader
 @import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.errorSummary
 @import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.saveButtons
+@import uk.gov.hmrc.plasticpackagingtax.registration.models.request.JourneyRequest
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.paragraphBody
+
 @this(
   govukLayout: main_template,
   saveButtons: saveButtons,
   govukInput: GovukInput,
   sectionHeader: sectionHeader,
   errorSummary: errorSummary,
-  formHelper: FormWithCSRF
+  formHelper: FormWithCSRF,
+  paragraphBody: paragraphBody,
+  govukFieldset: GovukFieldset,
 )
-@(form: Form[PhoneNumber])(implicit request: Request[_], messages: Messages)
 
-@phoneNumberField = @{form("value")}
+@(form: Form[PhoneNumber])(implicit request: JourneyRequest[_], messages: Messages)
 
-@govukLayout(
-  title = Title("primaryContactDetails.phoneNumber.title"),
-  backButton = Some(BackButton(messages("site.back"), pptRoutes.ContactDetailsEmailAddressController.displayPage(), messages("site.back.hiddenText")))) {
-  @errorSummary(form.errors)
 
-  @sectionHeader(messages("primaryContactDetails.sectionHeader"))
-
-  @formHelper(action = pptRoutes.ContactDetailsTelephoneNumberController.submit(), 'autoComplete -> "off") {
+@phoneNumberField = @{form("value")}
+@contactName = @{request.registration.primaryContactDetails.name.getOrElse("")}
```

Comment:
You could look up a default string to use if no name is present. `primaryContactDetails.fullName.default` has been added to `messages.en` for this purpose. It should probably be changed to "the main contact".

Reply:
Fixed